egs2/kising/svs1/run.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
```

Comment:
you may consider put the `dataset` argument here (and pass it to local_data_opts) so that users can still just use run.sh as the single entry.